tools/crmadmin.c

```diff
@@ -93,6 +95,11 @@ static GOptionEntry additional_options[] = {
       "\n                          operation failed",
       NULL
     },
+    { "node-types", 'T', 0, G_OPTION_ARG_STRING, &options.node_types,
+      "Node types to list (available options: all, member, pacemaker_remote,"
```

Comment:
I'd use "cluster" rather than "member" since that's how we refer to them in documentation